helix-term/src/commands.rs

```diff
@@ -3829,6 +3831,30 @@ fn goto_last_line_impl(cx: &mut Context, movement: Movement) {
     doc.set_selection(view.id, selection);
 }
 
+fn goto_column(cx: &mut Context) {
+    goto_column_impl(cx, Movement::Move);
+}
+
+fn extend_to_column(cx: &mut Context) {
+    goto_column_impl(cx, Movement::Extend);
+}
+
+fn goto_column_impl(cx: &mut Context, movement: Movement) {
+    if let Some(count) = cx.count {
+        let (view, doc) = current!(cx.editor);
+        let text = doc.text().slice(..);
+        let selection = doc.selection(view.id).clone().transform(|range| {
+            let line = range.cursor_line(text);
+            let line_start = text.line_to_char(line);
+            let target = line_start + count.get();
```

Comment:
This is treating the count as the number of chars. Instead we should treat it as the count of grapheme clusters - grapheme clusters are also used in the position statusline widget for example. See `helix_core::graphemes::nth_next_grapheme_boundary`